protocol/x/clob/keeper/keeper.go

```diff
@@ -233,17 +233,23 @@ func (k Keeper) InitializeNewGrpcStreams(ctx sdk.Context) {
 		allUpdates.Append(update)
 	}
 
-	k.SendOrderbookUpdates(allUpdates, true)
+	k.SendOrderbookUpdates(ctx, allUpdates, true)
 }
 
 // SendOrderbookUpdates sends the offchain updates to the gRPC streaming manager.
 func (k Keeper) SendOrderbookUpdates(
+	ctx sdk.Context,
 	offchainUpdates *types.OffchainUpdates,
 	snapshot bool,
 ) {
 	if len(offchainUpdates.Messages) == 0 {
 		return
 	}
 
-	k.GetGrpcStreamingManager().SendOrderbookUpdates(offchainUpdates, snapshot)
+	k.GetGrpcStreamingManager().SendOrderbookUpdates(
```

Comment:
don't really feel strongly at all but why don't we pass in the ctx instead of the blockheight and exec mode

Reply:
I think we can abstract away the context from the streaming manager. logic to get block height/exec mode doesn't really belong there.

let me know if you have a preference